lynx/common/actions/common_requirements.py

```diff
@@ -70,3 +70,6 @@ def any_object_on_square_has_all_given_tags(scene: Scene, position: Vector, give
                 return True
         return False
 
+    def has_something_in_inventory(scene: 'Scene', object_id: int) -> bool:
```

Comment:
`has_object_with_given_id_in_inventory`

Reply:
It implicates that we check if given object is in inventory.